test/lib/ufe/testSelection.py

```diff
@@ -118,3 +119,26 @@ def snFront(sn):
             cmds.redo()
             self.assertEqual(len(globalSn), 1)
             self.assertEqual(snFront(globalSn), i)
+
+    def testUfeSelect(self):
```

Comment:
This test exercises the ufeSelectCmd.  While useful, it does not demonstrate the bug, which is with the select command.